lib/Swagger/src/Discussion/Api/LegacyRedirectsApi.php

```diff
@@ -0,0 +1,317 @@
+                    break;
+            }
+
+            throw $e;
+        }
+    }
+
+    /**
+     * Operation getThreadRedirect
+     *
+     * Legacy Thread to Discussion Thread
+     *
+     * @param int $site_id The id of the site (required)
+     * @param int $thread_id Legacy Forum thread ID (required)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     * @return \Swagger\Client\Discussion\Models\LegacyRedirectResponse
+     */
+    public function getThreadRedirect($site_id, $thread_id)
+    {
+        list($response) = $this->getThreadRedirectWithHttpInfo($site_id, $thread_id);
+        return $response;
+    }
+
+    /**
+     * Operation getThreadRedirectWithHttpInfo
+     *
+     * Legacy Thread to Discussion Thread
+     *
+     * @param int $site_id The id of the site (required)
+     * @param int $thread_id Legacy Forum thread ID (required)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     * @return array of \Swagger\Client\Discussion\Models\LegacyRedirectResponse, HTTP status code,
+     * HTTP response headers (array of strings)
+     */
+    public function getThreadRedirectWithHttpInfo($site_id, $thread_id)
+    {
+        // verify the required parameter 'site_id' is set
+        if ($site_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $site_id when calling getThreadRedirect');
+        }
+        // verify the required parameter 'thread_id' is set
+        if ($thread_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $thread_id when calling getThreadRedirect');
+        }
+        // parse inputs
+        $resourcePath = "/{siteId}/redirect/thread/{threadId}";
+        $httpBody = '';
+        $queryParams = [];
+        $headerParams = [];
+        $formParams = [];
+        $_header_accept = $this->apiClient->selectHeaderAccept(['application/hal+json']);
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(['application/json']);
+
+        // path params
+        if ($site_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "siteId" . "}",
+                $this->apiClient->getSerializer()->toPathValue($site_id),
+                $resourcePath
+            );
+        }
+        // path params
+        if ($thread_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "threadId" . "}",
+                $this->apiClient->getSerializer()->toPathValue($thread_id),
+                $resourcePath
+            );
+        }
+        
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
+        } elseif (count($formParams) > 0) {
+            $httpBody = $formParams; // for HTTP post (form)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)